VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionAPI.java

```diff
@@ -144,6 +144,11 @@ public DiscussionThread followThread(DiscussionThread thread, boolean following)
         return discussionService.followThread(thread.getIdentifier(), followBody);
     }
 
+    public DiscussionThread readThread(DiscussionThread thread, boolean read)
```

Comment:
As I replied to the same comment on the task name, we should probably change the 'follow' and 'vote' methods as well.